infra-as-code/bicep/main.bicep

```diff
@@ -25,6 +25,9 @@ param publishFileName string = 'chatui.zip'
 @maxLength(123)
 param jumpBoxAdminPassword string
 
+@description('Specifies the SKU to be used by the Compute instance. Make sure you have the required ML quota available.')
+param computeInstanceSku string = 'Standard_D4ds_v4'

```

Comment:
In reference implementations, we typically never expose parameters that are not also exposed to the deployment instructions. Making something a bit more generic and flexible is fantastic for folks that need it, but we try to remove variability so that we can be repeatable in our tests.  Making "flexible" ARM templates isn't in scope for what we are trying to teach.  This one parameter being bubbled up is now oddly out of place with respect all the 10s of other parameters that could have been bubbled up.

I'm sorry that you ran into quota issues here.  It could have been in any of the resources, and we wouldn't bubble them all up like this every time someone bumped into one.

As common an instructional design approach we have for Azure Architecture Center reference architectures that this team creates, I'm sorry but I'll likely to change this this PR a bit.  Bare with me on some in bound suggestions here.